File: cryze_android_app/app/src/main/java/com/tencentcs/iotvideo/iotvideoplayer/IVideoRender.java
package com.tencentcs.iotvideo.iotvideoplayer;

import com.tencentcs.iotvideo.iotvideoplayer.codec.AVData;
import com.tencentcs.iotvideo.iotvideoplayer.codec.AVHeader;
/* loaded from: classes2.dex */
public interface IVideoRender {
    void onFrameUpdate(AVData aVData);

    void onInit(AVHeader aVHeader);

    void onPause();

    void onRelease();

    void onResume();
}

File: cryze_android_app/android/app/src/main/java/com/tencentcs/iotvideo/iotvideoplayer/IVideoRender.kt
package com.tencentcs.iotvideo.iotvideoplayer

import com.tencentcs.iotvideo.iotvideoplayer.codec.AVData
import com.tencentcs.iotvideo.iotvideoplayer.codec.AVHeader

// Must be in com.tencentcs.iotvideo.iotvideoplayer for JNI reasons
interface IVideoRender {
    fun onFrameUpdate(aVData: AVData?)

    fun onInit(aVHeader: AVHeader?)

    fun onPause()

    fun onRelease()

    fun onResume()
}
